examples/input_fodo.in

```diff
@@ -1,46 +1,45 @@
 ###############################################################################
 # Particle Beam(s)
 ###############################################################################
-beam.npart = 10000
+beam.npart = 100000
 beam.units = static
 beam.energy = 2.0e3
-beam.charge = 100.0e-12
+beam.charge = 0.0
 beam.particle = electron
 beam.distribution = waterbag
-beam.sigmaX = 1.0e-6
-beam.sigmaY = 1.0e-6
-beam.sigmaT = 1.0e-6
-beam.sigmaPx = 0.0
-beam.sigmaPy = 0.0
-beam.sigmaPt = 0.0
-beam.muxpx = 0.0
-beam.muypy = 0.0
+beam.sigmaX = 3.9984884770e-5
+beam.sigmaY = 3.9984884770e-5
+beam.sigmaT = 1.0e-3
+beam.sigmaPx = 2.6623538760e-5
+beam.sigmaPy = 2.6623538760e-5
+beam.sigmaPt = 2.0e-3
+beam.muxpx = -0.846574929020762
+beam.muypy = 0.846574929020762
 beam.mutpt = 0.0
 
 
 ###############################################################################
 # Beamline: lattice elements and segments
 ###############################################################################
-lattice.elements = quad1 drift1 quad2 drift2 sbend1
 
-quad1.type = quad
-quad1.ds = 1.0
-quad1.k = 4.0
+lattice.elements = drift1 quad1 drift2 quad2 drift3
 
 drift1.type = drift
-drift1.ds = 0.5
+drift1.ds = 0.25
 
-quad2.type = quad
-quad2.ds = 1.0
-quad2.k = 4.0
+quad1.type = quad
+quad1.ds = 1.0
+quad1.k = 1.0
 
 drift2.type = drift
 drift2.ds = 0.5
 
-sbend1.type = sbend
```

Comment:
Can you please add a 2nd input file in a follow up PR that uses sbends? Maybe a chicane? I just want to make sure we have that element covered in tests (totally ok to remove it here, because the file here reads FODO).

Reply:
Yes, I will add a file "input_chicane.in" in the next PR.